docs/ai/dotnet-ai-ecosystem.md

```diff
@@ -8,24 +8,62 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+The .NET ecosystem provides many powerful tools, libraries, and services to develop AI applications. .NET supports both cloud and local AI model connections, many different SDKs for various AI and vector database services, and other tools to help you build intelligent apps of varying scope and complexity.
 
 > [!IMPORTANT]
-> These SDKs and tools are built by a variety of sources. Not all SDKs are maintained by Microsoft. When considering an SDK, be sure to evaluate its quality, licensing, and support to ensure they meet your requirements. Also make sure you review each SDK's documentation for detailed version compatibility information.
+> Not all of the SDKs and services presented in this doc are maintained by Microsoft. When considering an SDK, make sure to evaluate its quality, licensing, support and compatibility to ensure they meet your requirements.
+
+## Semantic Kernel for .NET
+
+[Semantic Kernel](semantic-kernel-dotnet-overview.md) is an open-source SDK that enables AI integration and orchestration capabilities in your .NET apps. This SDK is generally the recommended AI orchestration tool for .NET apps that use one or more AI services in combination with other APIs or web services, data stores, custom code and beyond. Semantic Kernel benefits enterprise developers in the following ways:
 
-## Working with models
+- Streamlines integration of AI capabilities into existing applications to enable a cohesive solution for enterprise products.
+- Minimizes the learning curve of working with different AI models or services by providing abstractions that reduce complexity.
+- Improves reliability by reducing the unpredictable behavior of prompts and responses from AI models. You can fine-tune prompts and plan tasks to create a controlled and predictable user experience.
 
-Today, you can use .NET to access models built by OpenAI, using either the Azure OpenAI SDK or the Semantic Kernel. These models can be hosted by OpenAI or in Azure using the Azure AI services. Preview support is coming soon in Semantic Kernel to work with other models, and you can experiment today using open-source SDKs created by the .NET developer community.
+Visit the [Semantic Kernel documentation](/semantic-kernel/overview/) for more information.
 
-| NuGet package | Supported models | Maintainer or vendor | Link to docs |
+## .NET SDKs for building AI apps
+
+Many different SDKs are available for .NET to build apps with AI capabilities depending on the target platform or AI model. OpenAI models offer powerful generative AI capabilities, while other Azure AI Services provide intelligent solutions for a variety of specific scenarios.
+
+### .NET SDKs for OpenAI Models
```

Comment:
For now, the strongest story is around OpenAI models, but there will be others like Mistral that we want to promote. I think this section is fine for now, but we should also add the official OpenAI package: https://www.nuget.org/packages/OpenAI/